simtools/corsika/corsika_histograms.py

```diff
@@ -106,9 +98,15 @@ def _initialize_attributes(self):
         self.event_information = None
         self._individual_telescopes = None
         self._allowed_histograms = {"hist_position", "hist_direction", "hist_time_altitude"}
-        self._allowed_1D_labels = {"wavelength", "time", "altitude"}
-        self._allowed_2D_labels = {"counts", "density", "direction", "time_altitude"}
+        self._allowed_1d_labels = {"wavelength", "time", "altitude"}
+        self._allowed_2d_labels = {"counts", "density", "direction", "time_altitude"}
         self._header = None
+        self.hist_position = None
+        self.hist_direction = None
+        self.hist_time_altitude = None
+
+        self.read_event_information()
+        self._initialize_header()
```

Comment:
These changes here are breaking the test. I suggest returning it to the previous form and change only upper case D to lower case d.